metabrainz/payments/views.py

```diff
@@ -101,25 +102,25 @@ def check_editor():
 def complete():
     """Endpoint for successful payments."""
     if request.args.get("is_donation") == "True":
-        flash.success("Thank you for making a donation to the MetaBrainz Foundation. Your "
+        flash.success(gettext("Thank you for making a donation to the MetaBrainz Foundation. Your "
                       "support is greatly appreciated! It may take some time before your "
-                      "donation appears in the list due to processing delays.")
+                      "donation appears in the list due to processing delays."))
```

Comment:
Please keep indentation for strings like that consistent. It should be either
```python
flash.success(gettext("Thank you for making a donation to the MetaBrainz Foundation. Your "
                      "support is greatly appreciated! It may take some time before your "
                      "donation appears in the list due to processing delays."))
```
or
```python
flash.success(gettext(
    "Thank you for making a donation to the MetaBrainz Foundation. Your "
    "support is greatly appreciated! It may take some time before your "
    "donation appears in the list due to processing delays."
))
```

I prefer the second option.